<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Guetta</class>
 <widget class="QDialog" name="Guetta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1435</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guetta</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1421</width>
     <height>811</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>130</y>
       <width>261</width>
       <height>301</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>211</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Máximo nº keypoints</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_maxKeyPoints">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit__maxKeyPointsAlineamiento">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Máximo nº keypoints alienamiento</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_maxDistancia">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>211</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Máx distancia emparejamiento</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cambiar">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>240</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Cambiar</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cloud1: </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_narf">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>90</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>NARF</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1141</width>
       <height>791</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </widget>
    <widget class="QCheckBox" name="checkBox_acumular">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>760</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Acumular</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>450</y>
       <width>231</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Selección</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Distancia descriptor:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Distancia xyz:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_distanceDescriptor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_keyPoint1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>KeyPoint2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>KeyPoint1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_distanceXYZ">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_keyPoint2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioButton_surf">
     <property name="geometry">
      <rect>
       <x>1280</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SURF</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_sift">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SIFT</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox2">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>40</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cloud2:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox1">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>10</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Directorio:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_directorio">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>prueba1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_seleccionarDirectorio">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>111</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_procesar">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Procesar</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>661</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>591</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
